distsql/distsql.go

```diff
@@ -190,6 +190,12 @@ func (r *selectResult) getSelectResp() error {
 		if err != nil {
 			return errors.Trace(err)
 		}
+		if err := r.selectResp.Error; err != nil {
+			return terror.ClassTiKV.New(terror.ErrCode(err.Code), err.Msg)
```

Comment:
Should we process error code while tikv haven't implemented `Code`?

Reply:
Every error should have an error code. If it has not implemented in tikv, the error code is `ErrUnknown`(1105)